zmon-controller-ui/js/controllers/AlertDetailsCtrl.js

```diff
@@ -23,6 +23,9 @@ angular.module('zmon2App').controller('AlertDetailsCtrl', ['$scope', '$location'
         $scope.showActiveAlerts = true; // 1st tab, 1st button
         $scope.showAlertsInDowntime = false; // 1st tab, 2nd button
         $scope.showCheckResults = false; // 1st tab, 3rd button
+        $scope.alertDetailsSearch = { // entity filter
+            str: localStorageService.get('alertDetailsSearchStr') || ''
```

Comment:
Can you add a comment in the code? Otherwise it looks weird and I might be tempted to "clean up" the code later (forgetting the reason)...